crawler.py

```diff
@@ -84,21 +85,55 @@
 QUIET = False
 
 
+class SearchConfig:
+    def __init__(self):
+        with open("config.yaml", "r") as config_file:
+            for data in yaml.safe_load_all(config_file):
+                if "search" in data:
```

Comment:
Same here, the `for` loop can be removed, since there's now only one document left.